packages/typespec-azure-portal-core/test/decorators.test.ts

```diff
@@ -150,6 +188,34 @@ describe("TypeSpec-Azure-Portal-Core decorators test", () => {
     ]);
   });
 
+  it("@about with icon with wrong file extension", async () => {
+    const aboutTest = `
+      @test @about({
+        icon: {
+          filePath: "./icon.txt"
+        }
+      })`;
+    const diagnostics = await runner.diagnose(createTestSpec(undefined, aboutTest));
+    expectDiagnostics(diagnostics, {
+      code: "@azure-tools/typespec-azure-portal-core/invalid-type",
+      message: new RegExp("^@about.icon.filePath only allows svg file, current file:"),
```

Comment:
```suggestion
      message: /^@about.icon.filePath only allows svg file, current file:/,
```
you can do that for regex, makes it much more obvious and will be better for dealing with escapes